hail/python/test/hail/methods/test_impex.py

```diff
@@ -1995,14 +1996,6 @@ def test_key_by_after_empty_key_import(self):
         mt = mt.key_rows_by('Chromosome', 'Position')
         assert 0.001 < abs(0.50965 - mt.aggregate_entries(hl.agg.mean(mt.x)))
 
-    def test_key_by_after_empty_key_import(self):
```

Comment:
It deleted a full test?

Reply:
nope, me again, it complained because this test is named the same thing as the one directly above it, and since there wasn't an `assert` in the second one i figured it was unused